qiskit/circuit/quantumcircuit.py

```diff
@@ -1387,6 +1394,11 @@ def draw(self, output=None, scale=None, filename=None, style=None,
                               ax=ax,
                               initial_state=initial_state,
                               cregbundle=cregbundle)
+        if HAS_DISPLAY:
+            display(draw)
+            return None
+        else:
+            return draw
```

Comment:
If you were making this change why only on the draw method and not circuit_drawer itself?